WMFComponents/Sources/WMFComponents/Components/Shared/Feature Announcement/WMFFeatureAnnouncementView.swift

```diff
@@ -47,8 +47,23 @@ struct WMFFeatureAnnouncementView: View {
                                 .font(Font(WMFFont.for(.callout)))
                                 .foregroundColor(Color(appEnvironment.theme.text))
                         }
-
-                        if let image = viewModel.image {
+                        
+                        if let gifName = viewModel.gifName {
+                            ZStack {
+                                Image(gifName, bundle: .module)
```

Comment:
We may want to add an accessibility label on the image / gif in the feature announcement. It's reading out some strange stuff for me when I highlight it.